src/screens/Profile/Profile.jsx

```diff
@@ -0,0 +1,170 @@
+            <div className="description-wrapper">
+              <h3>{t("screens.profile.accountSettings.password.title")}</h3>
+              <p>{t("screens.profile.accountSettings.password.description")}</p>
+            </div>
+            <div className="button-wrapper">
+              <Button
+                block
+                uppercase
+                appearance="red"
+                onClick={openPasswordModal}
+              >
+                {t("screens.profile.accountSettings.password.title")}
+              </Button>
+            </div>
+          </div>
+          <div className="account-settings-fields border-bottom">
+            <div className="description-wrapper">
+              <h3>{t("screens.profile.accountSettings.email.title")}</h3>
+              <p>{t("screens.profile.accountSettings.email.description")}</p>
+            </div>
+            <div className="button-wrapper">
+              <Button block uppercase appearance="red" onClick={openEmailModal}>
+                {t("screens.profile.accountSettings.email.title")}
+              </Button>
+            </div>
+          </div>
+          <div className="account-settings-fields">
+            <div className="description-wrapper">
+              <h3>{t("screens.profile.accountSettings.delete.title")}</h3>
+              <p>{t("screens.profile.accountSettings.delete.description")}</p>
+            </div>
+            <div className="button-wrapper">
+              <Button
+                block
+                uppercase
+                appearance="red"
+                onClick={openDeleteAccountModal}
+              >
+                {t("screens.profile.accountSettings.delete.title")}
+              </Button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <Modal
+        title={t("screens.profile.modals.changePassword.heading")}
+        open={showPasswordModal}
+        onClose={closePasswordModal}
+      >
+        <h1 style={{ margin: "auto" }}>Hello World</h1>
+      </Modal>
+      <Modal
+        title={t("screens.profile.modals.changeEmail.heading")}
+        open={showEmailModal}
+        onClose={closeEmailModal}
+      >
+        <h1 style={{ margin: "auto" }}>Hello World</h1>
+      </Modal>
+      <Modal
+        title={t("screens.profile.modals.deleteAccount.heading")}
+        open={showDeleteAccountModal}
+        onClose={closeDeleteAccountModal}
+      >
+        <div className="deletion-modal">
+          <TextInput
+            required
+            placeholder={t("screens.profile.modals.deleteAccount.input")}
+            onChange={(value) => {
+              setDeleteConfirmation(value);
+              checkDeleteConfirmation(value);
+            }}
+            value={deleteConfirmation}
+          />
+          <Button
+            block
+            uppercase
+            appearance={"red"}
+            onClick={() => {
+              deleteUser();
+              handleLogout();
+            }}
```

Comment:
```suggestion
            onClick={async () => {
              await deleteUser();
              handleLogout();
            }}
```

Reply:
this should be solved with @zikowang fix

```
  const onDelete = useCallback(() => {
    deleteUser()
      .then((response) => {
        dispatch(signOut());
      })
      .catch((e) => {
        console.log(e);
      });
  }, [dispatch]);
  ```